crfs.go

```diff
@@ -147,6 +152,13 @@ var (
 	_ fspkg.HandleReader       = (*node)(nil)
 )
 
+// makedev packs a major and minor version how Linux expects.
```

Comment:
At least to me, this is pretty arcane.

I went looking for an authoritative source to link out to and found [`golang/sys/unix.MkDev`](https://github.com/golang/sys/blob/master/unix/dev_linux.go) which [references glibc](https://sourceware.org/git/?p=glibc.git;a=blob;f=bits/sysmacros.h;h=226023de7e167f89962f8d6a446d9feabff35bda;hb=HEAD). I don't know if it's reasonable to use `unix.Mkdev` directly, but we could link out to it to as documentation.